octoprint_mrbeam/software_update_information.py

```diff
@@ -54,6 +54,30 @@ class SWUpdateTier(Enum):
 BEAMOS_LEGACY_VERSION = "0.14.0"
 BEAMOS_LEGACY_DATE = date(2018, 1, 12)  # still used in the migrations
 
+FALLBACK_UPDATE_CONFIG = {
```

Comment:
Is this change related to the fix ?

Reply:
Yes I needed to refactor this so I am still able to give back the fallback update config